cmd/openqa-mon/tui.go

```diff
@@ -304,23 +332,31 @@ func (tui *TUI) Update() {
 		lines++
 	}
 	if tui.showHelp {
-		help := "?:Toggle help    r: Refresh    d:Toggle notifications    b:Toggle bell    +/-:Modify refresh time    p:Toggle pause"
+		help := "?:Toggle help  r:Refresh  d:Toggle notifications  b:Toggle bell  +/-:Modify refresh time  p:Toggle pause  <>:Page"
 		if len(tui.Model.HideStates) > 0 {
-			help += "    h:Toggle hide"
+			help += "  h:Toggle hide"
 		}
-		help += "    q:Quit"
+		help += "  q:Quit"
 		PrintLine(help, width)
 		lines++
 	}
-	fmt.Println()
-	lines++
-
 	offset := 0
-	maxHeight := height
+	pageHeight := height - 2
 	if tui.showStatus {
-		maxHeight -= 2
+		pageHeight -= 1
```

Comment:
```suggestion
		pageHeight--
```
Nitpicking: Let's stick with one increment/decrement style?